### PR TITLE
WIP:HMIS078-Postnatal Encounter Form

### PR DESCRIPTION
**Pull request by stansL**

### CODE REVIEW

**Comment by ningosi** [comment]
@stansL is this ready for review or there is still some work to be done?

File: omod/src/main/webapp/resources/htmlforms/078-IntegratedPostnatalRegister.xml [hunk 1]
```diff
@@ -1,591 +1,560 @@
...
           formShowIf="(visit.active || !visit.active) &amp;&amp; patient.person.dead==false &amp;&amp; sessionLocation.uuid=='e9bc61b5-69ff-414b-9cf0-0c22d6dfca2b'"
           formDisplayStyle="Standard"
-          formLabel="PNC Register Form">
-    <macros>
-        headerColor =#8fabc7
-        fontOnHeaderColor = white
-    </macros>
+          formLabel="PNC Encounter Form">
 
+    <style>
```
Comment (ssmusoke) [inline omod/src/main/webapp/resources/htmlforms/078-IntegratedPostnatalRegister.xml]: @stansL Look at the styling done by @moshonk cc @ningosi

Comment (ningosi) [inline omod/src/main/webapp/resources/htmlforms/078-IntegratedPostnatalRegister.xml]: @stansL can you find time to look into what @ssmusoke mentioned above?

Comment (stansL) [inline omod/src/main/webapp/resources/htmlforms/078-IntegratedPostnatalRegister.xml]: @ssmusoke @ningosi , taking a look at that, which form specifically?

**Pull request closed without merging**